content/docs/architecture/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Architecture"
+linkTitle: "Architecture"
+description: >
+  This section explains the basic architecture of the Openshift CI
```

Comment:
This doc specifically explains how the step registry and multi-stage works, but not the broader architecture of anything

Reply:
This is an index that creates the architecture setting on the sidebar, not something that is visible on the step registry page.

Reply:
Blaaaah.